examples/exec_with_res.c

```diff
@@ -135,9 +135,10 @@ int main(int argc, char *argv[])
 
 	input = 10; /* some random input value */
 	struct vaccel_arg read[1] = {
-		{.size = sizeof(input), .buf = &input}};
+		{.size = sizeof(input), .buf = &input, .argtype = 42}
```

Comment:
Thank you for the comments. Regarding argtypes, I would like to ask if there's any policy that I should follow. I mean, the purpose for which the argtypes were introduced in vaccel_arg, isn't it to maintain a "list" of predefined serializers / deserializers? So that when a user adds a known arg-type, the helper functions may use the predefined serializers?

Reply:
these arbitrary values -- this field is actually a placeholder for a forthcoming PR that would allow two things: (a) generic ser/deser functions embedded in vaccelrt, (b) vAccel resources to be used as arguments (read/write) for vAccel plugins. For now we just add it and these values are just for debug (we expect to see them through various debug snippets throughout the code). just ignore for now.